datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -118,7 +121,7 @@
     /**
      * A facade object providing access to the data stored by this <code>DataCapturingService</code>.
      */
-    private final PersistenceLayer<CapturingPersistenceBehaviour> persistenceLayer;
+    final PersistenceLayer<CapturingPersistenceBehaviour> persistenceLayer;
```

Comment:
required so that I can access the PersistenceLayer in the MovebisDataCapturingService.start() wrapper to load, delete, finish corrupted measurements.

I do not have the parameters there to create a new PersistenceLayer in this wrapper method.

Reply:
In Swift this would be ok, but in Java usually you should use a protected getPersistenceLayer accessor method for that purpose.